app/src/main/res/layout/security_dashboard.xml

```diff
@@ -1,11 +1,69 @@
 <?xml version="1.0" encoding="utf-8"?>
 <android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:background="@color/primaryBalticSea"
     tools:context="com.andela.art.securitydashboard.SecurityDashboardActivity">
 
-    <include layout="@layout/tool_bar"></include>
+    <include
+        android:id="@+id/mToolBar"
+        layout="@layout/tool_bar"/>
+
+    <de.hdodenhof.circleimageview.CircleImageView
+        android:id="@+id/profile_picture"
+        android:layout_width="130dp"
+        android:layout_height="130dp"
+        android:layout_marginTop="12dp"
+        android:src="@mipmap/ic_launcher_round"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/mToolBar"/>
+
+    <TextView
+        android:id="@+id/full_name"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="13dp"
+        android:text="@string/full_name"
+        android:textColor="#dadcdf"
+        android:fontFamily="@font/avenir_heavy"
+        android:textSize="22sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/profile_picture" />
+
+    <TextView
+        android:id="@+id/email_address"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/email"
+        android:textColor="#dadcdf"
```

Comment:
Since this text color `dadcdf` is used more than once (here and in the full_name TextView), it would better to store it's value in colors.xml such that incase have to change it in future, we change it from one place and it reflects everywhere it's used.

Reply:
Thanks, I have addressed this too.